assets/tests/components/__snapshots__/routeLadders.test.tsx.snap

```diff
@@ -48,90 +48,89 @@ exports[`renders a route ladder 1`] = `
       </button>
     </div>
   </div>
-  <svg
+  <div
```

Comment:
Does it matter that the class has moved from the SVG to a div?

Reply:
No, none of the properties are specific to the svg, only to the bounding box.